packages/webapp-libs/webapp-tenants/src/components/tenantMembersList/membershipEntry/membershipEntry.component.tsx

```diff
@@ -11,36 +12,71 @@ import {
   DropdownMenuSubTrigger,
   DropdownMenuTrigger,
 } from '@sb/webapp-core/components/dropdownMenu';
+import { Skeleton as SkeletonComponent } from '@sb/webapp-core/components/skeleton';
 import { TableCell, TableRow } from '@sb/webapp-core/components/table';
+import { useToast } from '@sb/webapp-core/toast';
 import { GripHorizontal, Hourglass, UserCheck } from 'lucide-react';
 import { indexBy, prop, trim } from 'ramda';
 import { useMemo } from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, useIntl } from 'react-intl';
 
 import { useTenantRoles } from '../../../hooks/useTenantRoles';
+import { useCurrentTenant } from '../../../providers';
+import { updateTenantMembershipMutation } from './membershipEntry.graphql';
 
 export type MembershipEntryProps = {
   className?: string;
   membership: TenantMembershipType;
+  refetch?: () => void;
```

Comment:
What do you think about changing this name to something more clear? Like `onAfterUpdate`?